frontend/src/components/StatusIcons.tsx

```diff
@@ -1,76 +1,41 @@
+import { CheckCircleIcon, ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons'
 import { Fragment, ReactNode } from 'react'
 
 export function StatusIcons(props: {
-    ready?: number
-    low?: number
-    medium?: number
-    high?: number
+    compliant?: number
+    violations?: number
     unknown?: number
-    readyTooltip?: string
-    lowTooltip?: string
-    mediumTooltip?: string
-    highTooltip?: string
+    compliantTooltip?: string
+    violationsTooltip?: string
     unknownTooltip?: string
 }) {
     const statuses: { key: string; count: number; icon: ReactNode; tooltip?: string }[] = []
 
-    if (props.ready !== undefined && props.ready) {
+    if (props.violations !== undefined && props.violations) {
         statuses.push({
-            key: 'ready',
-            count: props.ready,
-            icon: <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
-            tooltip: props.readyTooltip,
-        })
-    }
-
-    if (props.low !== undefined && props.low) {
-        statuses.push({
-            key: 'low',
-            count: props.low,
-            icon: <ChevronCircleDownIcon color="var(--pf-global--warning-color--100)" />,
-            tooltip: props.lowTooltip,
+            key: 'violations',
+            count: props.violations,
+            icon: <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />,
+            tooltip: props.violationsTooltip,
         })
     }
-    if (props.medium !== undefined && props.medium) {
+    if (props.compliant !== undefined && props.compliant) {
         statuses.push({
-            key: 'medium',
-            count: props.medium,
-            icon: (
-                <span style={{ position: 'relative' }}>
-                    <MinusCircleIcon color="var(--pf-global--warning-color--100)" />
-                    <MinusCircleIcon
-                        color="var(--pf-global--danger-color--100)"
-                        style={{ position: 'absolute', opacity: 0.5, left: 0, top: 2 }}
-                    />
-                </span>
-            ),
-            tooltip: props.mediumTooltip,
-        })
-    }
-    if (props.high !== undefined && props.high) {
-        statuses.push({
-            key: 'high',
-            count: props.high,
-            icon: <ChevronCircleUpIcon color="var(--pf-global--danger-color--100)" />,
-            tooltip: props.highTooltip,
+            key: 'compliant',
+            count: props.compliant,
+            icon: <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
+            tooltip: props.compliantTooltip,
         })
     }
 
     if (props.unknown !== undefined && props.unknown) {
         statuses.push({
             key: 'ready',
             count: props.unknown,
-            icon: <QuestionCircleIcon color="var(--pf-global--icon--Color--light)" />,
+            icon: <ExclamationTriangleIcon color="var(--pf-global--icon--Color--light)" />,
```

Comment:
I think the design has the yellow/orange color instead of the grey. 
`<ExclamationTriangleIcon color="orange"`